k8s_node_selector/README.MD

```diff
@@ -0,0 +1,39 @@
+# K8s Node Selector
+
+This is a simple example of an OPA policy which will apply a `nodeSelector` to each pod seen. 
+
+The purpose is to assign the workloads from certain `namespaces` to different `agentpools`.  
+
+## How to specify which `namespaces` map to which `agentpools`?
+
+Firstly the policy assumes that each agent pool has a label `agentpool: nameHere` assigned to all nodes within it. 
+
+Update the `namespaceToAgentPool` mapping in [`main.rego`](./main.rego) to specify which work ends up where. 
+
+You can also update the `ignoredNamespaces` to ensure namespaces, like `kube-system`, are ignored. 
+
```

Comment:
Could you expand the README a bit to include more information about what this directory is illustrating.  More specifics for someone who doesn't understand/remember why you use a nodeSelector and why it's important that pods have them.  Something like... 

K8s supports the concept of a nodeSelector to provide controls around which nodes each pod gets scheduled on.  Typically developers don't include nodeSelectors because <fill this in>, which means that an admin needs to write a mutating admission controller to tell k8s to automatically add them.  

In this example, we show how to use nodeSelectors to ensure that no two pods from different k8s namespaces ever run on the same node.  To do that, we will write OPA policies that map each namespace to different agentpools and insert nodeSelectors into every pod.

Reply:
Great call, I'll add this in shortly.

Reply:
Done, added more details on the use case this was developed to solve and gone into more detail. Let me know what you think (also fixed up spelling)